packages/react-aria-components/src/TextField.tsx

```diff
@@ -53,6 +54,7 @@ export interface TextFieldProps extends Omit<AriaTextFieldProps, 'label' | 'plac
 export const TextFieldContext = createContext<ContextValue<TextFieldProps, HTMLDivElement>>(null);
 
 function TextField(props: TextFieldProps, ref: ForwardedRef<HTMLDivElement>) {
+  let groupState = useContext(GroupStateContext);
```

Comment:
I just implemented this for `<TextField />` for now, but open to expand it to other field or action components.